test/unit_test_001.cpp

```diff
@@ -83,6 +86,50 @@ unittest(test_fletcher64)
 }
 
 
+unittest(test_FF_1)
+{
+  uint8_t x[1]  = { 0xFF };
+  uint16_t y[1] = { 0xFFFF };
+  uint32_t z[1] = { 0xFFFFFFFF };
+  
+  assertEqual(0, fletcher16(x, 1));
+  assertEqual(0, fletcher32(y, 1));
+  assertEqual(0, fletcher64(z, 1));
+}
+
+
+unittest(test_FF_2)
+{
+  uint8_t x[1]  = { 0xFF };
+  uint16_t y[1] = { 0xFFFF };
+  uint32_t z[1] = { 0xFFFFFFFF };
+
+  Fletcher16 FL16;
+  FL16.begin();
+  assertEqual(0, FL16.getFletcher());
+  FL16.begin(0xFF, 0);
+  assertEqual(0, FL16.getFletcher());
+  FL16.add(0xFF);
+  assertEqual(0, FL16.getFletcher());
+
+  Fletcher32 FL32;
+  FL32.begin();
+  assertEqual(0, FL32.getFletcher());
+  FL32.begin(0xFFFF, 0);
+  assertEqual(0, FL32.getFletcher());
+  FL32.add(0xFFFF);
+  assertEqual(0, FL32.getFletcher());
+
+  Fletcher64 FL64;
+  FL64.begin();
+  assertEqual(0, FL64.getFletcher());
+  FL64.begin(0xFFFFFFFF, 0);
+  assertEqual(0, FL64.getFletcher());
+  FL64.add(0xFFFFFFFF);
+  assertEqual(0, FL64.getFletcher());
```

Comment:
Similar to FL16. But this time I think only for `defined(ARDUINO_ARCH_SAMD)` it has no effect .